docs/get-started.md

```diff
@@ -191,8 +191,59 @@ kubectl delete crds -l component=velero
 kubectl delete -f examples/nginx-app/base.yaml
 ```
 
-[31]: expose-minio.md
+## Expose Minio outside your cluster
+
+When you run commands to get logs or describe a backup, the Velero server generates a pre-signed URL to download the requested items. To access these URLs from outside the cluster -- that is, from your Velero client -- you need to make Minio available outside the cluster. You can:
+
+- Change the Minio Service type from `ClusterIP` to `NodePort`.
```

Comment:
fyi, these don't work with KinD; using KinD, one must use `kubectl port-foward` due to how docker networking is set up. https://github.com/kubernetes-sigs/kind/issues/99 for details.

Reply:
OK - let's file a docs issue to add instructions for that case

Reply:
Created #1436